<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Huawei Traffic Analytics</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButton_DoHomography">
        <property name="text">
         <string>Do Homography</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_OpenImage1">
            <property name="text">
             <string>Open Image 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_DeleteLastPoint1">
            <property name="text">
             <string>Delete last point image 1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QtImageViewer" name="graphicsView_Image1"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_OpenImage2">
            <property name="text">
             <string>Open Image 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_DeleteLastPoint2">
            <property name="text">
             <string>Delete last point image 2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QtImageViewer" name="graphicsView_Image2"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtImageViewer</class>
   <extends>QGraphicsView</extends>
   <header>QtImageViewer</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
